### ISSUE TITLE
Existing skipped chapters (or others) leave empty folders behind

### ISSUE THREAD

**Issue by DJScias** [mansuf/mangadex-downloader#35]
### What happened ?

If you happen to download a manga and have certain chapters/volumes/etc.. already, it will mention it has cancelled the downlooad but it keeps an empty folder around for it. I mainly use it for cbz, so not sure if it's just for that save type.

The info log
![image](https://user-images.githubusercontent.com/439655/191307130-3ea8e02a-1025-4b43-9c81-1a7cdfe94c3c.png)

The empty folders that remain
![image](https://user-images.githubusercontent.com/439655/191308084-7accb400-d4b2-47bd-83da-5b4dfd20de9c.png)

### What did you expect to happen ?

No empty folders to be left behind when a chapter or anything is skipped.

### App version

mangadex-downloader v2.0.1 (https://github.com/mansuf/mangadex-downloader)
Python: 3.10.5
arch: x64
bundled executable: no

### Installation origin

PyPI (Python Package Index)

### Reproducible command
Obviously only reproducible if you have the 1st chapter of the manga already in the folder, then running the command again.
`mangadex-dl "xxxxxxxxxx" --save-as "cbz" --no-group-name --use-alt-details --start-chapter=1 --end-chapter=1`

**Comment by mansuf** [comment]
Fixed in https://github.com/mansuf/mangadex-downloader/commit/821461af805a2a471be46d8294a876aee98cf50e

You can try install development version if you want to try it early

```shell
py -m pip install git+https://github.com/mansuf/mangadex-downloader.git@821461af805a2a471be46d8294a876aee98cf50e
```

The issue was folder was created before checking if file (.cbz, cb7, etc) is exist or not 

https://github.com/mansuf/mangadex-downloader/blob/ec7a15baa26787c991244b08cf0495934c2690d7/mangadex_downloader/format/comic_book.py#L123-L133

Affected formats

- cbz
- cbz-volume
- epub
- epub-volume
- cb7
- cb7-volume

**Comment by mansuf** [comment]
Released in [v2.0.2](https://github.com/mansuf/mangadex-downloader/releases/tag/v2.0.2)
